src/main/java/com/hillel/items_exchange/service/AdvertisementService.java

```diff
@@ -116,6 +119,26 @@ public void remove(long id) {
         advertisementRepository.flush();
     }
 
+    public void setAdvertisementDefaultImage(Long advertisementId, Long imageId, User owner) {
```

Comment:
In my opinion you should rename `setAdvertisementDefaultImage` to `setDefaultImage`, because it's defined in **Advertisement**Service